### PR TITLE
GH-1945: [tree][navigator] The tree selection does not stretch horizontally anymore

### LINKED ISSUE #1945: [tree][navigator] The tree selection does not stretch horizontally anymore
selection-does-not-stretch-in-navigator

Also in the `Outline` view. I did not check the others.

Current:
screen shot 2018-05-24 at 09 45 49

Expected:
screen shot 2018-05-24 at 09 46 33

The breaking change: https://github.com/theia-ide/theia/commit/4996f9beefdf33be64236f5dba728ffcf3517e28#diff-83813e5ae443df2f48ba9a887da9a86dR26

### PR DESCRIPTION
**Pull request by jbicker**
Closes #1945.

### CODE REVIEW

**Comment by kittaakos** [comment]
I thought this was already fixed: https://github.com/theia-ide/theia/pull/1966

**Comment by jbicker** [comment]
@kittaakos: oh, yes. Sorry. I'll correct that.

File: packages/core/src/browser/tree/tree-widget.ts [hunk 1]
```diff
@@ -382,13 +382,14 @@ export class TreeWidget extends VirtualWidget implements StatefulWidget {
 
     protected renderNode(node: TreeNode, props: NodeProps): h.Child {
         const attributes = this.createNodeAttributes(node, props);
-        return h.div(attributes,
+        const contentEl = h.div({ className: "theia-TreeNodeContent" },
```
Comment (kittaakos) [inline packages/core/src/browser/tree/tree-widget.ts]: 1. Please use a [constant](https://github.com/theia-ide/theia/blob/c6c0e22176962853201a646ac632e072918c37d9/packages/core/src/browser/tree/tree-widget.ts#L34) for the class name.
 2. It is most likely just me, but `contentEl` means nothing more than `content`.

**Comment by kittaakos** [comment]
Awesome, the only outstanding TODO is to squash the commits and rewrite the commit message. The `Fixes GH-1965` is incorrect, it references another issue. Thanks!

**Comment by jbicker** [comment]
Ok...I guess now everything should be ok, right? :-)

**Comment by kittaakos** [comment]
> I guess now everything should be ok, right? :-)

❤️

**Pull request merged**